backend/test/resolvers/scholarship.js

```diff
@@ -49,6 +49,28 @@ describe("Scholarship Resolvers", () => {
           Scholarship.count().then(num => expect(num).to.equal(1));
         });
       });
+      it("Throws a helpful error if duplicate name scholarships are attempted", () => {
+        return createScholarship(
+          "",
+          { input: newScholarship },
+          { auth: { type: ADMIN } }
+        )
+          .then(() =>
+            createScholarship(
+              "",
+              { input: newScholarship },
+              { auth: { type: ADMIN } }
+            )
+          )
+          .then(() => {
+            assert.fail("Expected exception from promise");
+          })
+          .catch(error => {
+            expect(error.message).to.equal(
+              "Error: A Scholarship with that name already exists."
```

Comment:
I'm not using `expect.to.throw` because that doesn't correctly handle exceptions thrown in a promise. There is a library plugin that does a `expect.to.eventually.throw` for promises, but I didn't think it was worth getting when this workaround also gets the job done. I wouldn't be opposed to adding that library if anyone else thinks it's useful.